test/unit/sessions/video-session-handler.test.ts

```diff
@@ -553,7 +554,7 @@ describe('handlePropose', () => {
 describe('handleSessionInit', () => {
   it('should decorate the session', async () => {
     const parentHandler = jest.spyOn(BaseSessionHandler.prototype, 'handleSessionInit').mockResolvedValue(null);
-    const session: VideoMediaSession = {} as any;
+    const session: VideoMediaSession = { on: jest.fn() } as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9993633)
```suggestion
    const session: VideoMediaSession = { on: jest.fn() } as unknown;
```